subprojects/internal-performance-testing/src/main/groovy/org/gradle/profiler/GradleBuildInvokerPackageAccess.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.profiler;
+
+/**
+ * Needed since GradleBuildInvoker is package-private in gradle-profiler.
+ * This needs to be fixed in gradle-profiler.
+ */
+public class GradleBuildInvokerPackageAccess {
```

Comment:
This can be removed once https://github.com/gradle/gradle-profiler/pull/394 is merged and released.

Reply:
Do you have it on the radar to make a new release of Gradle profiler?